projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -5306,5 +5306,468 @@ public string RegistrationDate
                 }
             }
         }
+
+        /// <summary>Certifier name.</summary>
+        /// <value>the certifier's name</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.CertifierName = "Janet Seito";</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Certifier's Name: {ExampleBirthRecord.CertifierName}");</para>
+        /// </example>
+        [Property("Certifier Name", Property.Types.String, "Birth Certification", "Name of Certifier.", true, VR.IGURL.Practitioner, true, 6)]
+        [FHIRPath("Bundle.entry.resource.where($this is Practitioner).where(extension.value ='certifier')", "name")]
```

Comment:
Is the IG updated to show how we will use the extension.value set to "certifier"? Looking at the IG, I was thinking we would use the role field url to distinguish between the two. https://build.fhir.org/ig/HL7/fhir-bfdr/branches/master/StructureDefinition-Practitioner-birth-attendant.html